packages/app-root/src/app/groups/[groupId]/GroupStatsContainer.js

```diff
@@ -9,7 +9,13 @@ function GroupStatsContainer({
   groupId,
   joinToken
 }) {
-  const { adminMode, user } = useContext(PanoptesAuthContext)
+  const { adminMode, isLoading, user } = useContext(PanoptesAuthContext)
+
+  if (typeof window === 'undefined' || isLoading) {
+    return (
+      <p>Loading...</p>
```

Comment:
Just a side note that you can now use `<Loader/>` from lib-react-components for any loading UI if desired!